### PR TITLE
Sir 232

### PR DESCRIPTION
**Pull request by OttoVT**

### CODE REVIEW

File: src/Brokerage.Bilv1.Domain/Models/Blockchains/Blockchain.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using Newtonsoft.Json;
+
+namespace Brokerage.Bilv1.Domain.Models.Blockchains
+{
+    public sealed class Blockchain
```
Comment (KonstantinRyazantsev) [inline src/Brokerage.Bilv1.Domain/Models/Blockchains/Blockchain.cs:5]: Is this needed?

File: src/Brokerage.Bilv1.Domain/Models/Blockchains/BlockchainCapabilities.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Brokerage.Bilv1.Domain.Models.Blockchains
```
Comment (KonstantinRyazantsev) [inline src/Brokerage.Bilv1.Domain/Models/Blockchains/BlockchainCapabilities.cs:1]: Is entire this namespace needed?

File: src/Brokerage.Common/Persistence/Accounts/AccountRequisitesRepository.cs [hunk 3]
```diff
@@ -61,6 +61,22 @@ public AccountRequisitesRepository(DbContextOptionsBuilder<DatabaseContext> dbCo
                 .ToArray();
         }
 
+        public async Task<IReadOnlyCollection<AccountRequisites>> GetByAddressesAsync(string blockchainId, IReadOnlyCollection<string> addresses)
+        {
+            await using var context = new DatabaseContext(_dbContextOptionsBuilder.Options);
+
+            var query = context
+                .AccountRequisites
+                .Where(x => x.BlockchainId == blockchainId && addresses.Contains(x.Address));
```
Comment (KonstantinRyazantsev) [inline src/Brokerage.Common/Persistence/Accounts/AccountRequisitesRepository.cs:70]: Looks like there is no index for this request

File: src/Brokerage.Common/Persistence/Entities/BrokerAccountBalancesEntity.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        public decimal PendingBalance { get; set; }
+        public decimal ReservedBalance { get; set; }
+        public DateTimeOffset OwnedBalanceUpdateDateTime { get; set; }
+        public DateTimeOffset AvailableBalanceUpdateDateTime { get; set; }
+        public DateTimeOffset PendingBalanceUpdateDateTime { get; set; }
+        public DateTimeOffset ReservedBalanceUpdateDateTime { get; set; }
+    }
+
+    [Table(name: Tables.BrokerAccountBalancesUpdate)]
+    public class BrokerAccountBalancesUpdateEntity
```
Comment (KonstantinRyazantsev) [inline src/Brokerage.Common/Persistence/Entities/BrokerAccountBalancesEntity.cs:30]: Several types in the same file

File: src/Brokerage.Worker/MessageConsumers/FinalizeBrokerAccountCreationConsumer.cs [hunk 5]
```diff
@@ -105,7 +113,13 @@ public async Task Consume(ConsumeContext<FinalizeBrokerAccountCreation> context)
                 do
                 {
                     var result = await
-                        _brokerAccountRequisitesRepository.SearchAsync(brokerAccount.BrokerAccountId, 100, requisitesCursor, true);
+                        _brokerAccountRequisitesRepository.SearchAsync(
```
Comment (KonstantinRyazantsev) [inline src/Brokerage.Worker/MessageConsumers/FinalizeBrokerAccountCreationConsumer.cs:116]: It's not searching, but get all by BrokerAccount

File: src/Brokerage.Common/Persistence/Accounts/AccountRequisitesRepository.cs (hunk 3 above)
Comment (OttoVT) [reply src/Brokerage.Common/Persistence/Accounts/AccountRequisitesRepository.cs:70]: image

**Pull request merged**